### PR TITLE
CONSOLE-4187: Dev console: Use Metrics page from monitoring-plugin

### PR DESCRIPTION
**Pull request by zhuje**
**Description:** 
This PR aims to consolidate code so that the Administrator and Developer perspectives render the same UI, which the monitoring plugin will serve. 
This PR removes the files associated with the Dev Perspective > Observe > Metrics page. The monitoring plugin will now serve this page, PR for this is [openshift/monitoring-plugin#PR138](https://github.com/openshift/monitoring-plugin/pull/138). 

**Related JIRA Issues** 
[CONSOLE-4187](https://issues.redhat.com/browse/CONSOLE-4187) : Dev console: Use Metrics page from monitoring-plugin
[OU-258](https://issues.redhat.com/browse/OU-258): Dev console: Use Metrics page from monitoring-plugin
These are duplicate issues because the openshift/console requires the PR title to be associated with an OCP-owned JIRA issue. 

**Tandem PR:  openshift/monitoring-plugin PR#138** 
[openshift/monitoring-plugin PR#138](https://github.com/openshift/monitoring-plugin/pull/138): Dev console: Use Metrics page from monitoring-plugin

**Test:** 
To see the full feature, you will need to run both this openshift/console PR#14105 and this [openshift/monitoring-plugin PR#138](https://github.com/openshift/monitoring-plugin/pull/138).

This **openshift/console#PR14105** - removes the code associated with the Dev perspective > Observe > Metrics page. 
image

This **openshift/monitoring-plugin#PR138** - add the Dev perspective > Observe > Metrics page (a new UI renders, similar to Admin perspective > Observe > Metrics page). 
image

**Video Demo** 
Together (openshift/console#PR14105 & openshift/monitoring-plugin#PR138) should render like this: 
https://github.com/user-attachments/assets/a80b663f-c609-4f7c-9aca-c7edbd6bf5b0

**Note**
-The 403 Error displayed in the Dashboard is unrelated to this issue and has been reported here ([OCPBUGS-37511](https://issues.redhat.com/browse/OCPBUGS-37511?filter=-2)). The issue only occurs when running the /console in localhost.

### CODE REVIEW

File: frontend/packages/dev-console/src/components/monitoring/MonitoringPage.tsx [hunk 1]
```diff
@@ -42,12 +41,12 @@ export const PageContents: React.FC = () => {
...
-      // t('devconsole~Metrics')
-      nameKey: 'devconsole~Metrics',
-      component: ConnectedMonitoringMetrics,
-    },
+    // {
+    //   href: 'metrics',
+    //   // t('devconsole~Metrics')
+    //   nameKey: 'devconsole~Metrics',
+    //   component: ConnectedMonitoringMetrics,
+    // },
```
Comment (jgbernalp) [inline frontend/packages/dev-console/src/components/monitoring/MonitoringPage.tsx:49]:
```suggestion
```

**Comment by jgbernalp** [comment]
For qe and approvals
/assign @juzhao @opayne1 @reestr

**Comment by zhuje** [comment]
Squashed commits & yarn i18n in latest `git push --force`

**Comment by zhuje** [comment]
Approval from console member: 

/assign @karthikjeeyar

**Review by vikram-raj — APPROVED** [review]
looks good.

/approve

**Pull request merged**